switchbot/adv_parsers/blind_tilt.py

```diff
@@ -0,0 +1,22 @@
+"""Library to handle connection with Switchbot."""
+from __future__ import annotations
+
+
+def process_woblindtilt(
+    data: bytes | None, mfr_data: bytes | None, reverse: bool = True
+) -> dict[str, bool | int]:
+    """Process woBlindTilt services data."""
+
```

Comment:
`mfr_data ` can be likely be `None` if its anything like the other devices so it needs a guard here to return `None`